openmls/src/group/core_group/create_commit.rs

```diff
@@ -223,13 +223,13 @@ impl CoreGroup {
         let mut provisional_epoch = self.group_context.epoch();
         provisional_epoch.increment();
 
-        // Build MlsPlaintext
-        let mut mls_plaintext = MlsPlaintext::commit(
+        // Build MlsAuthContent
+        let mut content = MlsAuthContent::commit(
```

Comment:
Maybe just call it `commit` Makes creating the struct further down easier and is more descriptive.